tubular/base.py

```diff
@@ -241,50 +241,7 @@ def columns_check(self, X: pd.DataFrame) -> None:
                 raise ValueError(f"{self.classname()}: variable " + c + " is not in X")
 
 
-class BaseTwoColumnTransformer(BaseTransformer):
```

Comment:
Q: Is this still required here - https://github.com/lvgig/tubular/blob/e92f8dc95cb6016a4d730fd1dc6a9977a6b46e69/tubular/comparison.py#L9

Reply:
Actually, that's the next file, and can see you've changed it :)